components/k8s-model-server/http-proxy/server.py

```diff
@@ -125,25 +148,27 @@ def decode_b64_if_needed(data):
   else:
     return data
 
+
 def get_signature_map(model_server_stub, model_name):
   """ Gets tensorflow signature map from the model server stub.
 
-  Args:
-    model_server_stub: The grpc stub to call GetModelMetadata.
-    model_name: The model name.
+    Args:
+      model_server_stub: The grpc stub to call GetModelMetadata.
+      model_name: The model name.
 
-  Returns:
-    The signature map of the model.
+    Returns:
+      The signature map of the model.
   """
   request = get_model_metadata_pb2.GetModelMetadataRequest()
   request.model_spec.name = model_name
   request.metadata_field.append("signature_def")
   try:
-    response = model_server_stub.GetModelMetadata(request, MODEL_SERVER_METADATA_TIMEOUT_SEC)
+    response = model_server_stub.GetModelMetadata(
+      request, MODEL_SERVER_METADATA_TIMEOUT_SEC)
   except grpc.RpcError as rpc_error:
-    logging.exception("GetModelMetadata call to model server failed with code "
-                      "%s and message %s", rpc_error.code(),
-                      rpc_error.details())
+    logging.exception(
+      "GetModelMetadata call to model server failed with code "
```

Comment:
I preferred aligning on the "(" as it was before.

Reply:
This might be hard to achieve or possibly a conflicting `yapf` knobs' behavior.
On the other hand if we align to `(`, wouldn't that cause max line length to be consumed pretty quick when dealing with long strings like this one and we end up with too many lines.

I personally see this consistent because it's forcing the `4` space continuation policy 🙂